rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -236,14 +260,31 @@ def get_value():
             assert post_std != 0.0, post_std
             trainer.stop()
 
+    def test_ppo_legacy_config(self):
+        """Tests, whether the old PPO config dict is still functional."""
+        ppo_config = ppo.DEFAULT_CONFIG
+        # Expect warning.
+        print(f"Accessing learning-rate from legacy config dict: {ppo_config['lr']}")
+        # Build Trainer.
+        ppo_trainer = ppo.PPOTrainer(config=ppo_config, env="CartPole-v1")
```

Comment:
can we move env somehow to the ppo config?

Reply:
@avnishn that's a good one. In my prototype I tried to have the env name in the config (which is in principle allowed), but didn't have success in making this runnable. @sven1977 is it possible to move the env "down" to the config level? I'm also not sure what the implications are if you have to specify a custom env.

Reply:
It doesn't matter. It's agnostic to handle both: env defined in config or given in c'tor. c'tor will override what's in config.
For consistency, we should probably move it into config for good.

Reply:
... and not support this c'tor arg anymore.